test/TestPhaseTwoOpts.m

```diff
@@ -0,0 +1,52 @@
+classdef TestPhaseTwoOpts < matlab.unittest.TestCase
+    properties (TestParameter)
+        % Only test the 
+        piece_nlp_strategy = {'TNLP', 'BNLP_integer', 'BNLP_Gradient1', 'BNLP_Gradient2'};
```

Comment:
huh. BNLP was the dominant option i was using for a long time. 
I am wondering to the other three still work :D

Reply:
ok, all works